lib/util.js

```diff
@@ -371,6 +372,23 @@ function _exceptionWithHostPort(...args) {
   return new ExceptionWithHostPort(...args);
 }
 
+let dotenv;
+
+/**
+ * Loads the .env file from the given path.
+ * @param {string | undefined} path
+ * @returns {void}
+ */
+function loadEnvFile(path) {
+  dotenv ??= internalBinding('dotenv');
+  if (path != null) {
+    validateString(path, 'path');
+    dotenv.load(path);
+  } else {
+    dotenv.load();
+  }
```

Comment:
With the current implementation, if you use `node --env-file=not-exist.env index.mjs`, you will encounter an error if the .env file does not exist.
I think we should apply the same approach here as well

Reply:
I think we failed to follow the general use case. Dotenv doesn't throw an error instead warns the user: https://github.com/motdotla/dotenv/blob/master/lib/main.js#L228

I recommend not throwing an error.